tests/baker_recipes/works.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+
+from model_bakery.recipe import Recipe, foreign_key
+
+from .publishers import journal
+
+# Recipes for the `works.Article` objects
+
+# Real article data:
+# See: https://www.sciencedirect.com/science/article/abs/pii/S0378381211001282
+
+article = Recipe(
+    "works.Article",
+    title=(
+        "Liquid–liquid phase equilibrium of (piperidinium-based ionic liquid + "
+        "an alcohol) binary systems and modelling with NRHB and PCP-SAFT"
+    ),
+    year=2012,
+    doi="10.1016/j.fluid.2011.03.005",
+    journal=foreign_key(journal),
+    volume="305 (1)",
+    pages="43-52",
+)
+
+
+# Recipes for the `works.Book` objects
+
+# Real book data:
+# See: https://pubs.acs.org/doi/10.1021/bk-2002-0818.ch001
+
+book = Recipe(
+    "works.Book",
+    publishing_house__abbr="ACS",
+    title="Ionic Liquids: Industrial Applications for Green Chemistry",
+    chapter_title="Green Chemistry and Ionic Liquids: Synergies and Ironies",
+    doi="10.1021/bk-2002-0818.ch001",
+    year=2002,
+    pages="2-14",
+    isbn="978-0-8412-3789-6",
+)
+
+
+# Recipes for the `works.Patent` objects
+
+# Real patent data:
+# See: https://patents.google.com/patent/US20040262578
+
+patent = Recipe(
+    "works.Patent",
+    title="Ionic Liquids",
+    number="US20040262578A1",
+    date_applied=datetime.date(2004, 3, 31),
+    date_published=datetime.date(2004, 12, 30),
+)
+
+__all__ = ["article", "book", "patent"]
```

Comment:
Insert extra line before `__all__`.

```suggestion

__all__ = ["article", "book", "patent"]
```